cognitedata/reveal
# fix: reject malformed / invalid 360 images from being created

#### Type of change

![Bug](https://img.shields.io/badge/Type-Bug-red) 

#### Jira ticket :blue_book:


https://cognitedata.atlassian.net/browse/REV-720

## Description :pencil:


Now preloads file descriptions and filters out events that don't map to valid files.

## How has this been tested? :mag:



on 3d-test we currently have celanese1 dataset which has a chunck of invalid events, and celanese2 where all are invalid.


## Test instructions :information_source:



load the `celanese1` or `celanese2` dataset and see a warning being thrown in the console.
You can also compare the number of entities returned with current master (which includes invalid 360 images).

## Checklist :ballot_box_with_check:
